src/Tizen.Uix.InputMethod/Tizen.Uix.InputMethod/InputMethodEditor.cs

```diff
@@ -2139,5 +2141,75 @@ public static void SetFloatingDragEnd()
                 throw InputMethodExceptionFactory.CreateException(error);
             }
         }
+
+        /// <summary>
+        /// Notifies the changed language of the input panel to the the associated text input UI control.
+        /// </summary>
+        /// <privilege>
+        /// http://tizen.org/privilege/ime
+        /// </privilege>
+        /// <remarks>
+        /// LanguageRequestedCallback is raised after this API is called when the App requests changed language information.
+        /// </remarks>
+        /// <exception cref="InvalidOperationException">
+        /// This can occur due to the following reasons:
+        /// 1) The application does not have the privilege to call this function.
+        /// 2) The IME main loop has not yet started.
```

Comment:
The following sentence is better:

        /// 2) The IME main loop has not started yet.

Reply:
Fixed.

Reply:
The English reviewer changed from "isn't started yet" to "has not yet started".

Reply:
Fixed.